dlib/svm/auto.cpp

```diff
@@ -101,6 +97,101 @@ namespace dlib
 
         return df;
     }
+
+// ----------------------------------------------------------------------------------------
+
+    normalized_function<multiclass_linear_decision_function<linear_kernel<matrix<double,0,1>>, unsigned long>>
+    auto_train_multiclass_svm_linear_classifier (
+        std::vector<matrix<double,0,1>> x,
+        std::vector<unsigned long> y,
+        const std::chrono::nanoseconds max_runtime,
+        bool be_verbose
+    )
+    {
+        const auto labels = select_all_distinct_labels(y);
+        for (const auto label : labels)
+        {
+            const auto num_samples = sum(mat(y) == label);
+            DLIB_CASSERT(num_samples >= 6,
```

Comment:
Shouldn't these all be 3 instead of 6?  Since there are only 3 folds?

Reply:
right... copy paste...

Reply:
Oh this is copied from what I did. :confused:

Reply:
Oh but that other function uses 6 fold cross validation.  So it needs 6.  This one only needs 3 though :)

Reply:
I copied the assert only...

Reply:
Yeah no worries.  I thought for a moment that somehow 3 fold CV needs 6 and was like "whaaaa???" lol

Reply:
yes, that's what I meant, that I copied the assert, without modifying the value properly, my bad, I will push the change.

Reply:
Yeah, no problem :)